### PR TITLE
Dev alberto

### PR DESCRIPTION
**Pull request by AlbertoCervantes30**
The merge that I am pulling are Unit Tests to update and improve the "Learn More", and the "Team" files.

Learn More file, index file tested

Team file, index file tested
This request is updated. The old request had the Learn More index file in another file, I've deleted it and added the correct file in the correct place.

### CODE REVIEW

**Review by ThatZiv — APPROVED** [review]
Looks good!

**Comment by XLRA** [comment]
Updated Alberto's dev branch as it wasn't updated to the latest changes in the development branch.

File: src/components/LearnMore/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```
Comment (PierreT20) [inline src/components/LearnMore/index.test.tsx:1]: Descriptive Test Blocks: Each test block (it function) has a clear and descriptive name that outlines what the test is checking. This makes the tests easy to understand and maintain.
Component Rendering: The render(<LearnMore />); statement in each test block correctly sets up the component for testing, ensuring that the DOM nodes are available for assertions.

File: src/components/Team/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (PierreT20) [inline src/components/Team/index.test.tsx:1]: Descriptive Test Cases: The test descriptions clearly define what each test aims to verify within the Team component. This helps in understanding the purpose of each test quickly.
Effective Assertions: The tests use toBeInTheDocument() to verify the presence of specific texts and components. This is a robust way to check if the rendering behaves as expected.
Testing Multiple Instances: The checks for multiple instances of SingleTeam components, social links, and images are good examples of how to handle multiple elements. This is crucial for ensuring that all expected parts of the component render correctly.

**Pull request merged**